content/docs/command-reference/exp/show.md

```diff
@@ -111,7 +111,9 @@ metric or param.
   experiment names.
 
 - `--show-json` - prints the command's output in easily parsable JSON format,
-  instead of a human-readable table.
+  instead of a human-readable table .
+
+- `--show-csv` - Print output in csv format instead of a human-readable table.
```

Comment:
Print -> prints

csv -> CSV

`table .` -> table.`

Reply:
also, for consistency why don't we do `the command's output`

Reply:
```suggestion
- `--show-csv` - prints the command's output in CSV format instead of a human-readable table.
```